addon/mixins/model-records/columns.js

```diff
@@ -22,12 +25,12 @@ export default Ember.Mixin.create(RecordTypeMixin, {
   }),
 
   filteredColumns: filter('columns', function(name) {
-    var modelName            = get(this, 'model-record.name');
-    var adminIncludedColumns = this.admin.includedColumns;
-    var adminExcludedColumns = this.admin.excludedColumns;
-    var includedColumns      = this.includedColumns;
-    var excludedColumns      = this.excludedColumns;
-    var allowColumn          = true;
+    const modelName            = get(this, 'model-record.name');
+    const adminIncludedColumns = this.admin.includedColumns;
+    const adminExcludedColumns = this.admin.excludedColumns;
+    const includedColumns      = this.includedColumns;
+    const excludedColumns      = this.excludedColumns;
+    var allowColumn            = true;
```

Comment:
Avoid `var` if you can. Prefer `const` for value references not expected to be mutated, else use `let`